core/broadleaf-framework/src/main/java/org/broadleafcommerce/core/util/service/ResourcePurgeServiceImpl.java

```diff
@@ -136,6 +156,71 @@ public void purgeCarts(final Map<String, String> config) {
         LOG.info(String.format("Cart purge batch processed.  Purged %d from total batch size of %d, %d failures cached", processedCount, batchCount, cartPurgeErrors.size()));
     }
 
+    @Override
+    public void purgeOrderHistory(Class<?> rootType, String rootTypeIdValue, Map<String, List<DeleteStatementGeneratorImpl.PathElement>> depends, final Map<String, Integer> config) {
+
+        if (LOG.isDebugEnabled()) {
+            LOG.debug("Purging historical orders");
+        }
+
+        String enablePurge = env.getProperty("enable.purge.order.history");
+
+        if (!Boolean.parseBoolean(enablePurge)) {
+            LOG.info("Save protection. Purging history is off. Please set property enable.purge.order.history to true.");
+            return;
+        }
+
+        Integer daysCount = config.get(OLDER_THAN_DAYS.toString());
+        Integer batchSize = config.get(BATCH_SIZE.toString());
```

Comment:
This is batch size defined in line 94. Instead of static import, import whole class and do following
```
 Integer daysCount = config.get(PurgeOrderHistoryVariableNames.OLDER_THAN_DAYS.toString());
 Integer batchSize = config.get(PurgeOrderHistoryVariableNames.BATCH_SIZE.toString());
```